2022-07_sprint/schedule.md

```diff
@@ -1,16 +1,17 @@
 (2022_07/schedule)=
 # Sprint schedule
-
 :::{include} schedule_table.md
 :::
 
-:::
 
-## Schedule with Timezones
+:::{dropdown} Open dropdown for spreadsheet with times in more time zones
+:icon: clock
 
-[![schedule with timezones](../_static/2022_07/2022_07_timezones.png)](https://docs.google.com/spreadsheets/d/1oO9OhsTh9PFcqKI7Exj_ZcH0GvunQVHtM4aqUh0kjTk/edit?usp=sharing)
+<iframe src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSJ_x4V63aji-WmarsgEyj9eS_Cp0DB5m647TZdNjYj03nPQq0u0Q6eKPGhHrw-8Ugp16KhAg-AgLkK/pubhtml?widget=true&amp;headers=false" height="320" width="750"></iframe>
```

Comment:
The issue was not with removing the dropdown but with the non-pairing of opening and closing colon fences. If you prefer the image without dropdown we should not reintroduce the dropdown. And this won't render right because there is no closing colon fence for the dropdown.

Reply:
With the missing closing fence, everything under the dropdown ends up inside it, because it is not closed. https://pymc-data-umbrella--89.org.readthedocs.build/en/89/2022-07_sprint/schedule.html

Reply:
@OriolAbril 
Can you do "suggested changes" here so we can accept?

Reply:
it wasn't possible to do suggest changes so I did the change directly in a new commit